tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/ILanguage.cs

```diff
@@ -26,6 +26,7 @@ public interface ILanguage
             string testName,
             string arguments,
             bool profile,
-            object context);
+            object context,
+            int iteration);
```

Comment:
```suggestion
            object context);
```

Unfortunately, I don't think passing the `iteration` is actually sufficient to disambiguate, because the same test can be run with different arguments.  So the profiles for `download(10KB)` would conflict with `download(10MB)`.  If you want a counter, I would increment a static integer like `Java.cs`.  Another option would be to try to include the arguments in the filename, but this might get very long (e.g. `download-size-10240-parallel-64-1`).

In practice, people usually collect profiles for one test and one set of arguments at a time, but I wouldn't want conflicts if someone did try to collect profiles for multiple arguments in a single run.

Reply:
I was thinking of this long name, but it'll be too long. Will do.
Let's revert at a later point if we feel like it is too long.